code/game/objects/items/shields.dm

```diff
@@ -236,3 +236,20 @@
 	throwforce = 15 //Massive pice of metal
 	w_class = WEIGHT_CLASS_HUGE
 	item_flags = SLOWS_WHILE_IN_HAND
+
+/obj/item/shield/riot/implant
+	name = "riot tower shield"
+	desc = "A massive shield that can block a lot of attacks, can take a lot of abuse before braking."
+	item_state = "metal"
+	icon_state = "metal"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	block_chance = 30 //May be big but hard to move around to block.
+	force = 10
+	slowdown = 1
+	transparent = FALSE
+	item_flags = SLOWS_WHILE_IN_HAND
+
+/obj/item/shield/riot/implant/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(attack_type == PROJECTILE_ATTACK)
+		final_block_chance = 60 //Massive shield
```

Comment:
Can we not?

Reply:
?

Reply:
Actually, for something that requires surgery and eats up an arm slot for the much more useful utility of the toolset arm, it's probably not bad to have it possess a distinct advantage over standard equipment. The toolspeeds of the implant tools is really high if I remember correctly. 10% doesn't realistically make THAT much of a difference, but it is a no-drop shield.

Also, do people really abuse Alexander that much? It's alcohol content is fairly high, you'd need a cyberliver to really make the most of it.

Reply:
The problem with Alexander drink is that you can abuse it a lot. And 70% projectile block is to much for me to even.
40% block with the drink is ok in my eyes as its still less then normal shields
As for it being better then normal gear, it has a slowdown and less damage so that getting the combo if no drop + shield is better then the implant.
This is not ment to be the `Oh shit install this into my body` every round

Reply:
No longer a issue